integration.md

```diff
@@ -0,0 +1,130 @@
+>
+> As a policy-controlled feature, the fullscreen API can specify that when
+> disabled, `document.fullscreenEnabled` must return `false`, and
+> `Element.requestFullscreen()` must fire a `fullscreenerror` event, and return
+> a promise which rejects with a `TypeError` exception.
+
+If an existing feature has no mechanism for reporting failure, or has never been
+allowed to fail in the past, then it may be dangerous to turn it into a
+policy-controlled feature, where it may start to fail on pages that were always
+able to assume that it would not.
+
+In this case, features can take advantage of the concept of
+[feature-policy-awareness](https://wicg.github.io/feature-policy/#feature-policy-aware)
+to define different behaviour when disabled in a document written to be aware of
+Feature Policy (and hence the possiblity that the API could fail) and in a
+document written without that knowledge.
+
+> Example:
+>
+> document.write API has existed on the web since roughly the dawn of
+> JavaScript, and has no failure modes. This means that if a page contains a
+> script like this:
+>
+>     <html>
+>       <head>
+>         <script>document.write('<script src="important-security-feature.js"></scr'+'ipt>')</script>
+>         ...
+>
+> then an attacker could embed the site in a frame with the `document.write`
+> feature disabled, and bypass the critical security script.
+>
+> To avoid this, if disallowed in a feature-policy-aware document, calling
+> `document.write()` will fire a `SecurityError` exception on the `document`
+> object. If disallowed in a feature-policy-oblivious document, calling
+> `document.write()` feature must cause the current page to be navigated to an
+> error page which provides the option to open the page that violated the policy
+> in a new top-level browsing context.
+
+## Allow versus Deny features
+
+In some cases, when defining a new policy-controlled feature, it may not be
+obvious whether allowing or restricting an action should be considered the
+feature.
+
+In a case like this, consider that Feature Policy tries to guarantee that, once
+disabled in a document, a feature -- however defined -- cannot be reenabled in
+that document or any of its children.
+
+> As an example, if a browser was considering a feature which would block the
+> loading of any images which appeared to be pictures of cats (as a
+> productivity measure,) and wanted to make this a policy-controlled feature,
+> then the developers would have a choice -- whether to define the new feature
+> as "block cat pictures", or as "load cat pictures".
+>
+> If the goal is to block cat pictures, such that they cannot be loaded into a
+> document at all, then defining the feature as "load cat pictures" makes
+> sense, and the feature would be given a default allowlist of `[*]`. This way,
+> cat pictures would be allowed by default on the entire web, but once disabled
+> in a document, would not be available anywhere in that document, even in
+> child frames.
```

Comment:
I think there are other reasons to express the feature in terms of the ability to do something:
-Double negatives are confusing and error-prone
-Consistency is a good thing to try to encourage across features.